actions.py

```diff
@@ -112,11 +112,33 @@ def toggle_show_on_player(self):
         elif self.data.player_mode == 'next':
             self.video_driver.next_player.toggle_show()
 
+    def increase_seek_time(self):
+        for idx, i in enumerate(self.data.settings['sampler']['SEEK_TIME']['options']):
```

Comment:
this looks good! let me make a suggestion in the interest of educating (and coz im excited to be collaborating after many solo sessions wishing someone would be questioning what im writing!)

have you come across the % (modulo) function before ? it is a good way to 'wrap around' like what your if/elif statement is doing, but with the added advantage that you do not need to know (hardcode) the size of the list you are wrapping around, so if i was to for example add another step size to my settings code i wouldnt also need to update this function,

perhaps something like `new_index = (current_index + 1) % length_of_list`